packages/testsuite/cypress/e2e/test-configuration-subsystem-health.cy.ts

```diff
@@ -0,0 +1,39 @@
+describe("TESTS: Configuration => Subsystems => Health", () => {
+
+  const address = ["subsystem", "health"];
+  const configurationFormId = "model-browser-model-browser-root-form";
+  const securityEnabled = "security-enabled";
+
+  let managementEndpoint: string;
+
+  before(() => {
+    cy.startWildflyContainer().then((result) => {
+      managementEndpoint = result as string;
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Navigate to Health subsystem", () => {
```

Comment:
Could you please rename it to "toggle security-enabled" as that is the matter of this test?

Reply:
sure! renamed to "toggle security-enabled".